.circleci/config.yml

```diff
@@ -26,7 +26,9 @@ commands:
       - run: go build ./...
   go_test:
     steps:
-      - run: go test -v ./...
+      - run:
+          no_output_timeout: 15m
```

Comment:
I'm not entirely sure why we receive no output for more than `10m` and I don't see how to check how close we were to the default 10m timeout before, but this all seems to hang on `e2etest` package tests.

A cleaner solution here would be adding `t.Log` around operations which are likely to take long time - e.g. installation of Terraform, `init` (installation of providers) etc. but I avoided doing that for now, until the TF installation is decoupled per https://github.com/hashicorp/terraform-exec/issues/109